graphistry/umap_utils.py

```diff
@@ -197,8 +205,11 @@ def umap_lazy_init(
             self._umap = umap_engine.UMAP(**umap_kwargs)
             self.umap_initialized = True
             self.engine = engine_resolved
-            self.suffix = suffix
-
+            if (engine_resolved=='cuml'):# and (float(umap_engine.__version__[:5])<22.06):
+                self.suffix = float(umap_engine.__version__[:5]) ##
+            else:
+                self.suffix = suffix
+                
```

Comment:
I think field suffix is reserved for some other use? No need to save the version name, can dynamically look.up...

Reply:
i tried looking it up again -- but `self._umap().__module__` does not work

Reply:
since `self._umap = umap_engine.UMAP(**umap_kwargs)`
`self._umap().__module__` returns
TypeError: 'UMAP' object is not callable

Reply:
agreed trying to hide 22.04 in suffix brings more trouble than its worth with type check errors

Reply:
`suffix` is reserved for other stuff, so would just entirely remove your use of `suffix`, and revert back to the behavior before this